### PR TITLE
Fixed issue point tray when metadata is null

### PR DESCRIPTION
**Pull request by goyal1092**
## Description
In Recent updates we made metadata undefined value check strict and that resulted in excluding null from check.
I have added specific check for null as well as used `?.` just as precaution so we do not get this issue.

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-550

## How should a reviewer test it locally
Vist point tray for sifar profile (profile_id =6)
With this change it won't throw error. but for any other production or staging deploy preview it will throw erorr.

## Checklist

- [x]  🚀 is the code ready to be merged and go live?
- [x]  🛠 does it work (build) locally
- [x] 👩‍🎨 does the design match the [Demo](https://wazimap-ng-v1.webflow.io/demo)

### Pull Request

- [x]  📰 good title
- [x]  📝good description
- [x]  🔖 issue linked
- [x]  📖 changelog filled out
- [x] commit messages are meaningful

### Code Quality

- [x]  🚧 no commented out code
- [x]  🖨 no unnecessary logging
- [x]  🎱 no magic numbers

### Testing

- [ ]  ✅ added (appropriate) unit tests
- [ ]  💢 edge cases in tests were considered
- [ ]  ✅ ran tests locally & are passing

### CODE REVIEW

File: src/js/elements/point_tray/category.js [hunk 1]
```diff
@@ -18,10 +18,10 @@ export class Category extends Component {
     fixData(data) {
         data = {...data}
 
-        if (data.metadata === undefined)
+        if (data.metadata === undefined || data.metadata === null)
             data.metadata = {}
 
-        if (data.metadata.source === undefined)
+        if (data.metadata?.source === undefined)
```
Comment (michaelglenister) [inline src/js/elements/point_tray/category.js:24]: Is the question mark supposed to be there?

**Review by michaelglenister — COMMENTED** [review]
The fix looks good. For tests, if something broke and we fix it there should be a way to test that it doesn't break again right

If we want to get this fix out quickly we can create a separate card to discuss or add a test

File: src/js/elements/point_tray/category.js (hunk 1 above)
Comment (goyal1092) [reply src/js/elements/point_tray/category.js:24]: yup `?.` is an operator in ES6
if metadata is null or undefined it won't throw error if it didn't find source.. 
It will be just undefined
added it as a precaution

**Comment by goyal1092** [comment]
@michaelglenister Updated PR. Please review

**Review by michaelglenister — APPROVED** [review]
Approved
